### PR TITLE
Display a message when there is a problem with the default module

### LINKED ISSUE #15975: Encryption can be enabled via occ even if the app itself is not enabled
### Steps to reproduce

1. Install 8.1
2. From occ, enable encryption

### Expected behaviour

Either the app gets enabled as part of the request to enable encryption or an error is thrown to inform the user that the encryption app is not yet enabled.

### Actual behaviour

Encryptoin gets enabled without enabling the app or throwing an error to inform the user that the encryption app is not yet enabled.

```
root@ubuntu-14-lts:/var/www/html/owncloud# sudo -u www-data php occ app:list
Enabled:
 - activity: 1.2.3
 - admin_audit: 0.6
 - enterprise_key: 0.1.0
 - files: 1.1.9
 - files_locking:
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.1
 - files_sharing_log: 0.5
 - files_texteditor: 0.4
 - files_trashbin: 0.6.2
 - files_versions: 1.0.5
 - files_videoviewer: 0.1.3
 - gallery: 0.6.0
 - provisioning_api: 0.2
 - templateeditor: 0.1
 - updater: 0.4
 - windows_network_drive: 0.1.1
Disabled:
 - encryption
 - encryption_dummy
 - external
 - files_antivirus
 - files_drop
 - files_external
 - files_ldap_home
 - firewall
 - firstrunwizard
 - objectstore
 - sharepoint
 - user_external
 - user_ldap
 - user_shibboleth
 - user_webdavauth
root@ubuntu-14-lts:/var/www/html/owncloud# sudo -u www-data php occ encryption:enable
Encryption enabled
Default module: OC_DEFAULT_MODULE
root@ubuntu-14-lts:/var/www/html/owncloud# sudo -u www-data php occ app:list       Enabled:
 - activity: 1.2.3
 - admin_audit: 0.6
 - enterprise_key: 0.1.0
 - files: 1.1.9
 - files_locking:
 - files_pdfviewer: 0.7
 - files_sharing: 0.6.1
 - files_sharing_log: 0.5
 - files_texteditor: 0.4
 - files_trashbin: 0.6.2
 - files_versions: 1.0.5
 - files_videoviewer: 0.1.3
 - gallery: 0.6.0
 - provisioning_api: 0.2
 - templateeditor: 0.1
 - updater: 0.4
 - windows_network_drive: 0.1.1
Disabled:
 - encryption
 - encryption_dummy
 - external
 - files_antivirus
 - files_drop
 - files_external
 - files_ldap_home
 - firewall
 - firstrunwizard
 - objectstore
 - sharepoint
 - user_external
 - user_ldap
 - user_shibboleth
 - user_webdavauth
```

### Server Host info:

Ubuntu 14.04
PHP 5.5.9

### Server configuration:

ownCloud Enterprise Edition 8.1 alpha 2 (daily) Build:2015-04-30T03:08:25+00:00 4209757d61685346b9b3eb476035b4e86597ed64

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #15975

@jnfrmarks @schiesbn @th3fallen @DeepDiver1975 
### No module:

no-module
### Invalid module:

invalid-module
### Valid module:

valid-module

### CODE REVIEW

File: core/command/encryption/enable.php [hunk 1]
```diff
@@ -50,9 +57,22 @@ protected function execute(InputInterface $input, OutputInterface $output) {
...
 			$this->config->setAppValue('core', 'encryption_enabled', 'yes');
-			$output->writeln('Encryption enabled');
+			$output->writeln('<info>Encryption enabled</info>');
 		}
+		$output->writeln('');
 
-		$output->writeln('Default module: ' . $this->config->getAppValue('core', 'default_encryption_module', 'OC_DEFAULT_MODULE'));
+		$modules = $this->encryptionManager->getEncryptionModules();
+		if (empty($modules)) {
+			$output->writeln('<error>No encryption module is loaded</error>');
```
Comment (nickvergessen) [inline core/command/encryption/enable.php]: Maybe we should extend this message a bit?
` ... Please go to the app list and install an encryption app e.g. "ownCloud Default Encryption Module"`

Comment (DeepDiver1975) [inline core/command/encryption/enable.php]: to me the message is enough as is.

**Comment by schiesbn** [comment]
looks good to me :+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**